cfme/tests/control/test_compliance.py

```diff
@@ -62,7 +62,9 @@ def compliance_vm(request, provider_key, provider_crud):
         appliance.configure(setup_fleece=True)
         vm = Vm(appliance.vm_name, provider_crud)
     # Do the final touches
-    with appliance.ipapp:
+    appliance.ipapp.browser_steal = True
+    with appliance.ipapp as appl:
+        appl.set_session_timeout(86400)
```

Comment:
It occurs to me that we should probably be doing this as a result of opening the browser, logging in, or some other appropriate event. I also think it's likely that using the `IPAppliance` context manager interface should be all you need, and we should automatically set `browser_steal` when using it, or accept `browser_steal` as an argument to the context manager.